UI/data/locale/en-US.ini

```diff
@@ -1531,3 +1523,44 @@ YouTube.Errors.rateLimitExceeded="You are sending messages too quickly."
 # Browser Dock
 YouTube.DocksRemoval.Title="Clear Legacy YouTube Browser Docks"
 YouTube.DocksRemoval.Text="These browser docks will be removed as deprecated:\n\n%1\nUse \"Docks/YouTube Live Control Room\" instead."
+
+# TODO: Double check if all of these are actually used
+SceneCollections.Title="Manage Scene Collections"
+SceneCollections.Search="Type to search"
+SceneCollections.New="Create New Collection..."
+SceneCollections.Import="Import Collection..."
+SceneCollections.Open="Open"
+SceneCollections.Export="Export..."
+SceneCollections.Duplicate="Duplicate..."
+SceneCollections.Delete="Delete..."
+SceneCollections.Rename="Rename..."
+SceneCollections.Sort.LastUsed="Last used"
+SceneCollections.Sort.Name="Name"
+
+SceneCollections.New.Title="Create Scene Collection"
+SceneCollections.GenericNamePrompt.Text="Please enter the name of the scene collection"
+SceneCollections.Duplicate.Title="Duplicate Scene Collection"
+SceneCollections.Duplicate.Default="%1 (Duplicate)"
+SceneCollections.Rename.Title="Rename Scene Collection"
+SceneCollections.Delete.Title="Delete Scene Collection"
+SceneCollections.Delete.Text="Do you want to delete the scene collection \"%1\"?"
+SceneCollections.Export.Title="Export Scene Collection"
+SceneCollections.BulkDuplicate.Title="Duplicate Scene Collections"
+SceneCollections.BulkDuplicate.Text="Do you want to duplicate selected %1 scene collections?"
```

Comment:
Are you sure you wish to duplicate %1 scene collections?

Reply:
Done